nodeadm/test/e2e/run.sh

```diff
@@ -18,6 +18,7 @@ for CASE_DIR in $(ls -d test/e2e/cases/*); do
   CONTAINER_ID=$(docker run \
     -d \
     --rm \
+    --cpus "4" \
```

Comment:
what's this about?

Reply:
This is for the e2e testing, the calculation for kubeReserved.cpu is depending on the return value of `runtime.NumCPU()`, which might change in different setup(my local docker engine use 8), thus trying to use this as a pin to get consistent e2e test result

Reply:
Ah I see. `runtime.NumCPU()` will give you the number of cores available to the current process, the same as `GOMAXPROCS`, not the total number of physical/virtual cores on the system.

It looks like `kubelet` gets this information (which is reported as the allocatable resources on the node) with cadvisor: https://github.com/kubernetes/kubernetes/blob/2fb99eb9e81ec9fc92d8e694227772928f2eecc5/pkg/kubelet/cadvisor/util.go#L42

That ultimately comes from here in cadvisor: https://github.com/google/cadvisor/blob/35dd21c64581692aae5fcaf3a470e8a0fe54fcf6/utils/sysinfo/sysinfo.go#L203

I don't love the idea of adding a dependency on all of cadvisor, but we can either copy the relevant impl or pull it in as a last resort.

Reply:
copy and strip down please. cadvisor is `ktlo`